libs/components/action-bars/src/lib/modules/summary-action-bar/summary-action-bar.component.html

```diff
@@ -7,6 +7,29 @@
   }"
 >
   <div class="sky-summary-action-bar-actions">
+    <div aria-live="polite">
+      @if (formErrors) {
+        @if (formErrors.length > 0) {
+          <div class="sky-summary-action-bar-errors sky-margin-stacked-lg">
```

Comment:
Should we use css with the correct token instead of the `sky-margin-stacked-lg` here and then only apply the margin if there is content? Also - what happens if someone creates a summary action bar with no actions and an error then pops up?